cmd/describe/cluster/cmd.go

```diff
@@ -837,6 +842,47 @@ func getAuditLogForwardingStatus(cluster *cmv1.Cluster) string {
 	return auditLogForwardingStatus
 }
 
+func getClusterRegistryConfig(cluster *cmv1.Cluster) string {
+	var output string
+	if cluster.RegistryConfig().RegistrySources() != nil {
+		registryResources := cluster.RegistryConfig().RegistrySources()
```

Comment:
let's get them in a dedicated section like Network

Reply:
👍 Added https://github.com/openshift/rosa/pull/2472/files#diff-15a63c5c54a2463b4203ef656bf435bf0e1a4f9725eecbb2015d503d0dc298b9R501